radix-engine-common/src/types/type_identifier.rs

```diff
@@ -0,0 +1,6 @@
+use crate::crypto::Hash;
+use crate::Sbor;
+use sbor::LocalTypeIndex;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Sbor)]
+pub struct TypeIdentifier(pub Hash, pub LocalTypeIndex);
```

Comment:
Minor - but would prefer this to have named fields so the hash can be named clearly; and a Rust doc explaining what this type is for.